frontend/javascripts/oxalis/controller/combinations/tool_controls.ts

```diff
@@ -780,6 +784,171 @@ export class QuickSelectTool {
   static onToolDeselected() {}
 }
 
+export class LineMeasurementTool {
+  static DOUBLE_CLICK_TIME_THRESHOLD = 600;
+  static getPlaneMouseControls(_planeId: OrthoView): any {
```

Comment:
`getPlaneMouseControls` of the area measurement tool doesn't accept that parameter. maybe make these interfaces consistent?